types/mongoose/index.d.ts

```diff
@@ -2407,15 +2407,17 @@ declare module "mongoose" {
          * Adds a discriminator type.
          * @param name discriminator model name
          * @param schema discriminator model schema
+         * @param value the string stored in the `discriminatorKey` property
          */
-        discriminator<U extends Document>(name: string, schema: Schema): Model<U>;
+        discriminator<U extends Document>(name: string, schema: Schema, value?: string): Model<U>;
 
         /**
          * Adds a discriminator type.
          * @param name discriminator model name
          * @param schema discriminator model schema
+         * @param value the string stored in the `discriminatorKey` property
          */
-        discriminator<U extends Document, M extends Model<U>>(name: string, schema: Schema): M;
+        discriminator<U extends Document, M extends Model<U>>(name: string, schema: Schema, value?: string): M;
```

Comment:
Could you add a test or two that covers this function signature change?

Reply:
Sorry, im new to this repo - do you mean mongoose code, or testing the typings?

Reply:
does 8d6981d8ada94d745a8941438a8d01661e776fab do what you asked?

Reply:
Yes, that's exactly it!